job/job.go

```diff
@@ -99,22 +100,24 @@ func CreateFromIP(ip net.IP) (Job, error) {
 		return Job{}, err
 	}
 
-	hd := d.Hardware()
-	hwID := hd.HardwareID()
+	if os.Getenv("DATA_MODEL_VERSION") == "1" {
```

Comment:
How about 
```
if os.Getenv("DATA_MODEL_VERSION != "" {
    return j, nil
}

old new code here
```
instead

Reply:
I can do that

Reply:
IDK what you're talking about, there's definitely a `!= "1"` up there. All the other uses of DATA_MODEL_VERSION have been in a switch statement and here want to do early return instead.

Reply:
🤔  that was definitely something different a second ago, heh. cool, will do.